core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/MachineConfig.java

```diff
@@ -20,13 +20,6 @@
  */
 public interface MachineConfig {
 
-  /**
-   * Name of the attribute from {@link #getAttributes()} which if present sets memory limit of the
-   * machine in bytes. If memory limit is set in environment specific recipe this attribute should
-   * override value from recipe.
-   */
-  String MEMORY_LIMIT_ATTRIBUTE = "memoryLimitBytes";
```

Comment:
-1 to move this into Machine object since it is needed on configuration step to assign a specific amount of RAM to a machine